frontend/src/pages/pipelines/global/experiments/ArchiveExperimentModal.tsx

```diff
@@ -0,0 +1,44 @@
+import { StackItem } from '@patternfly/react-core';
+import React from 'react';
+import ArchiveModal from '~/concepts/pipelines/content/ArchiveModal';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import { ExperimentKFv2 } from '~/concepts/pipelines/kfTypes';
+
+interface ArchiveExperimentModalProps {
+  experiment: ExperimentKFv2;
+  onCancel: () => void;
+  isOpen: boolean;
+}
+
+export const ArchiveExperimentModal: React.FC<ArchiveExperimentModalProps> = ({
+  experiment,
+  onCancel,
+  isOpen,
+}) => {
+  const { api } = usePipelinesAPI();
+  const onSubmit = React.useCallback(
+    () => api.archiveExperiment({}, experiment.experiment_id),
+    [api, experiment.experiment_id],
+  );
+  return (
+    <ArchiveModal
+      isOpen={isOpen}
+      resource={experiment}
+      onCancel={onCancel}
+      onSubmit={onSubmit}
+      title="Archive experiment?"
+      testId="archive-experiment-modal"
+    >
+      <>
+        <StackItem>
+          Archiving the <b>{experiment.display_name}</b> experiment will archive its runs and
+          disable all scheduled runs.
+        </StackItem>
+        <StackItem>
+          The <b>{experiment.display_name}</b> can be restored, but its runs and scheduled runs will
```

Comment:
This is not grammatically correct. "The experiment can be restored" would be a more appropriate sentence structure. "experiment" could be tacked on at the end of the display name, but that's already happening in the opening statement and feels unnecessary to repeat.

Reply:
<img width="1708" alt="Screenshot 2024-03-18 at 10 35 19 AM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/5c57c2e5-2cce-439a-95d7-1ee07e96fc91">

I am following the mockup here, I can change it though if you think that's more grammatically correct.

Reply:
Yeah, it ain't great English. Another option is to just strip the "The" before the experiment name. I'm willing to discuss with design if you want to get them involved.

Reply:
I will go with the second option for now, which is stripping the `The` before the name.

Reply:
I think going with the second option sounds fine. However, I'm confused as to why we have the "Type [experiment name] to confirm archiving:" part here. I didn't think we were including this type verification in archive modals (see the 'Experiments' section of the content doc: https://docs.google.com/document/d/11SquEHVcyMVtuejCZxgAnFoY43GagYjQBhbfaI17Pn0/edit)